src/goInstallTools.ts

```diff
@@ -496,9 +513,10 @@ export function checkLanguageServer(): boolean {
 		return false;
 	}
 
-	let langServerAvailable = getBinPath('go-langserver') !== 'go-langserver';
+	const languageServerOfChoice = getAlternateLanguageServer(latestGoConfig) || 'go-langserver';
+	const langServerAvailable = path.isAbsolute(getBinPath('go-langserver'));
```

Comment:
shouldn't we check here for `languageServerOfChoice`

Reply:
`getBinPath` will internally look up alternateTools and do the right thing